lib/mobility/backends/active_record/key_value.rb

```diff
@@ -198,17 +203,6 @@ def translation_for(locale, **)
         translation
       end
 
-      module DestroyKeyValueTranslations
```

Comment:
I realized that there's really no point using a module for this since including it multiple times will continue to trigger the `included` hook (the whole point was to only trigger the hook the first time, but that's not how Ruby works).